molecule/default/molecule-with-systemd.yml

```diff
@@ -15,6 +15,17 @@ lint: |
   ansible-lint
   flake8
 platforms:
+   # Amazon Linux 2023 does not (yet?) offer docker-compose or the
+   # Docker compose plugin:
+   # - cgroupns_mode: host
```

Comment:
Add helpful link:
```suggestion
   # Amazon Linux 2023 does not (yet?) offer docker-compose or the
   # Docker compose plugin:
   # https://github.com/amazonlinux/amazon-linux-2023/issues/186
   # - cgroupns_mode: host
```

Reply:
@jmorrowomni https://github.com/cisagov/ansible-role-pca-gophish-composition/commit/6c4681c982ea5c9fdbb69ab6465f3203a2da9df5 doesn't include my suggestion above for `molecule/default/molecule-with-systemd.yml` - it only includes my suggestion in `meta/main.yml`.

Reply:
I swore I clicked both - done now